client/lib/gsuite/index.js

```diff
@@ -34,6 +46,33 @@ function canDomainAddGSuite( domainName ) {
 	return ! ( hasInvalidSuffix || includesBannedPhrase || isGSuiteRestricted() );
 }
 
+/**
+ * Formats price gievn cost and currency
+ *
+ * @param {Number} cost - cost
+ * @param {String} currencyCode - currency code to format with
+ * @param {Object} options - options containing precision
+ * @returns {Boolean} - Returns a formatted price
+ */
+function formatPrice( cost, currencyCode, options = {} ) {
+	if ( undefined !== options.precision ) {
+		cost = applyPrecision( cost, options.precision );
+	}
+
+	return formatCurrency( cost, currencyCode, cost % 1 > 0 ? {} : { precision: 0 } );
+}
+
+/**
+ * Gets the formatted annual cost
```

Comment:
I think it would be a good idea to add examples to those functions so anyone looking at them can see what sort of output they can expect.